templates/history/history_list.html

```diff
@@ -4,7 +4,7 @@
 {% block title %}{% trans "History for" %} {{ device.name }}{% endblock %}
 
 {% block content %}
-    {% include "snippets/subpagination.html" with parent=device.pk listurl='device-history-list' %}
+    {% include "snippets/subpagination.html" with parent=device.pk listurl='history-list' %}
```

Comment:
`subpgaination.html` calls `{% url listurl parent page %}`. But `history-list` takes three parameters: `content_type` is missing.